docs/source/asr/example_training_asr_on_kinyarwanda_mcv.rst

```diff
@@ -0,0 +1,641 @@
+    {"audio_filepath": "test/common_voice_rw_24256935.wav", "text": "umunani", "up_votes": 2, "down_votes": 0, "age": NaN, "gender": NaN, "accents": NaN, "client_id": "974d4876e99e7437183c20f9107053acc9e514379d448bcf00aaaabc0927f5380128af86d39650867fa80a82525110dfc40784a5371c989de1a5bdf531f6d943", "duration": 3.24, "pred_text": "umunani"}
+
+Word Error Rate (WER) and Character Error Rate (CER)
+####################################################
+
+As soon as we have a manifest file with `text` and `pred_text` we can measure the quality of predictions of our model.
+
+.. code-block:: bash
+
+    # Calculate WER
+    python ${NEMO_ROOT}/examples/asr/speech_to_text_eval.py \
+      dataset_manifest=test_with_predictions.json \
+      use_cer=False \
+      only_score_manifest=True
+
+    # Calculate CER
+    python ${NEMO_ROOT}/examples/asr/speech_to_text_eval.py \
+      dataset_manifest=test_with_predictions.json \
+      use_cer=True \
+      only_score_manifest=True
+
+
+Evaluation of NVIDIA's Kinyarwanda checkpoints
+##############################################
+
+If you run inference and evaluation of Nvidia's published Kinyarwanda models, you should get metrics like these:
+
++----------------------------------+-------+-------+
+| Model                            | WER % | CER % |
++==================================+=======+=======+
+| stt_rw_conformer_ctc_large       | 18.22 | 5.45  |
++----------------------------------+-------+-------+
+| stt_rw_conformer_trasducer_large | 16.19 | 5.7   |
++----------------------------------+-------+-------+
+
+Error analysis
+##############
+
+Still, even WER of 16% is not as good as we usually get for other languages trained with NeMo toolkit, so we may want to look at the errors that the model makes to better understand what's the problem.
+
+We can use `Speech Data Explorer <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/tools/speech_data_explorer.html>`_ to analyze the errors.
+
+If we run
+
+.. code-block:: bash
+
+    python ${NEMO_ROOT}/tools/speech_data_explorer/data_explorer.py <your manifest file>
+
+it will start a local server, and provide a http address to open from the browser.
+In the UI we can see the model predictions and their diff with the reference, and also we can listen to the corresponding audio. We also can sort the sentences by descending WER and look through the top of them.
+
+The error analysis showed several problems concerning the Kinyarwanda dataset:
+
+* Noisy multi-speaker records (e.g. common_voice_rw_19830859.wav)
+* Bad quality of record (e.g. common_voice_rw_24452415.wav)
+* Orthographic variability related to space/no space/apostrophe
+    * *kugira ngo / kugirango*
+    * *nkuko / nk'uko*
+    * *n iyo / n'iyo*
+* Multiple orthographic variants for foreign words
+    * *telefoni / telephone*
+    * *film / filime*
+    * *isiraheli / israel*
+    * *radio / radiyo*
+    * *kongo / congo*
```

Comment:
Reading this post, it should really be a tutorial inside asr as a notebook, not a doc. The code here will not be tested, and is prone to break in the future. Nor is it executable for the user. 

Fyi @vsl9

Reply:
I understand your points, but need to wait for @vsl9 to comment.
This tutorial/doc was intended for some upcoming MCV event. There is a difficulty with executing this in a notebook because of restrictions on automatic download of mcv data.

Reply:
is it possible to download part of data via HF? Kinyarwanda is 50+Gb - too heavy to download

Reply:
I believe they have a streaming mode so you can use that and subset the data